src/comments/comments.service.ts

```diff
@@ -17,20 +18,25 @@ export class CommentsService {
     @Inject(forwardRef(() => ArticlesService))
     private articleService: ArticlesService,
   ) {}
-  async create(user: User, comments: CreateCommentDto, articleId: number) {
+  async create(user: User, comments: CreateCommentDto) {
+    const articleId = comments.articleId;
     const article = await this.articleService.findArticle(articleId);
     const createComment = this.commentRepository.create({
       ...comments,
       author: user,
       articles: article,
     });
```

Comment:
muốn reply comment thì ở CreateCommentDto thêm commentId
check nếu có  commentId thì findOne by id
chổ createComment => parent: comment
bỏ dòng 30-> 32

Reply:
hàm save bao gồm cả update và create
nên làm đơn giản là 
const commented = this.commentRepository.save({
      ...comments,
      author: user,
      articles: article,
})

return commented;